emacs/merlin.el

```diff
@@ -1493,6 +1493,8 @@ loading"
                              "-look-for" merlin-locate-preference)))
     (unless result
       (error "Not found. (Check *Messages* for potential errors)"))
+    (when (stringp result)
+      (user-error "%s" result))
     (unless (listp result)
       (error "%S" result))
```

Comment:
I am not sure why you added a case instead of simply replacing `(error ...` occurences by `(user-error ...` ?

Reply:
I think it should be sufficient, it look like such "hard errors" are already thrown inside `merlin-call`.

Reply:
Right, patch now updated accordingly.